Covid-19Radar/Covid19Radar
# Fix minSdkVersion to 23

## Purpose

* Fix minSdkVersion to 23 ref #319
* Refactoring uses-feature description to one line (VS2019 default)

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
```

* Test the code

```
Build Covid19Radar.Android and check this build install with fail on Nexus 7(2012), Android 5.1.0
```

## What to Check
Verify that the following are valid
* Install with fail on Nexus 7(2012), Android 5.1.0

## Other Information
